.github/config/android-cobalt-arm.json

```diff
@@ -13,9 +13,7 @@
         "gpu_unittests",
         "gin_unittests",
         "blink_unittests",
-        "content_shell",
-        "system_webview_apk",
-        "system_webview_shell_apk",
+        "cobalt_apk",
```

Comment:
instead of adding it here, will be better to add it to //cobalt:gn_all  . YOu can say (is_android) { dep+= ""cobalt_apk")